README.md

```diff
@@ -31,4 +59,155 @@ from `package.json` and `package-lock.json`
     }
     ```
 
-1. Run `npm install` to download the library and include it in `package-lock.json`
\ No newline at end of file
+1. Run `npm install` to download the library and include it in `package-lock.json`
+
+
+# API
+
+## `BraveAlerter` class
+
+The main class of this library. It is used to send single alerts or to start alert sessions with the responders.
+
+### constructor(getAlertSession, getAlertSessionByPhoneNumber, alertSessionChangedCallback, asksIncidentDetails, getReturnMessage)
+
+**getAlertSession (async function(sessionId)):** function that returns the AlertSession object with the given sessionId
+
+**getAlertSessionByPhoneNumber (async function(toPhoneNumber)):** function that returns the AlertSession object for the most recent unfinished session with the given phone number
+
+**alertSessionChangedCallback (async function(alertSession)):** function that will be called whenever an alertSession's values change; should be used to update the session in the DB
+
+**asksIncidentDetails (boolean):** `true` if alert sessions should ask for incident details, `false` otherwise
+
+**getReturnMessage (function(fromAlertState, toAlertState)):** function that returns the message to return when there is a transition from `fromAlertState` to `toAlertState`
```

Comment:
Would it make sense to replace the second 'return' with 'send'? Use of 'return' twice is a bit confusing. 

getReturnMessage returns a message when there is an AlertState transition; however, as I refer to in another comment below, how does it deal with invalid user input that doesn't result in a state transition? Does getReturnMessage send some helper text for the user?